arrows/ocv/image_container.cxx

```diff
@@ -36,13 +36,30 @@
 #include "image_container.h"
 
 #include <arrows/ocv/mat_image_memory.h>
+#include <opencv2/imgproc/imgproc.hpp>
 
 using namespace kwiver::vital;
 
 namespace kwiver {
 namespace arrows {
 namespace ocv {
 
+image_container
+::image_container(const cv::Mat& d, ColorMode cm)
+  : data_(d)
+{
+  if(cm != RGB)
```

Comment:
Eep! I know we are not using spaces consistently project-wide, but can we at least not introduce new inconsistency within a single diff hunk? (Here and some other places in this PR...)